### PR TITLE
Fix bug that tablet failed to be committed when no data is loaded

### PR DESCRIPTION
**Pull request by morningman**
ISSUE: #2063

### CODE REVIEW

**Review by chaoyli — DISMISSED** [review]

File: fe/src/main/java/org/apache/doris/catalog/Catalog.java [hunk 1]
```diff
@@ -2959,7 +2959,12 @@ public void addPartition(Database db, String tableName, AddPartitionClause addPa
 
                 // check partition name
                 if (olapTable.getPartition(partitionName) != null) {
-                    throw new DdlException("Partition " + partitionName + " already exists");
+                    if (singlePartitionDesc.isSetIfNotExists()) {
+                        LOG.info("add partition[{}] which already exists", partitionName);
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/Catalog.java]: Do we need this log?

Comment (morningman) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: I will change it to debug

Comment (morningman) [inline fe/src/main/java/org/apache/doris/catalog/Catalog.java]:
```suggestion
                        LOG.debug("add partition[{}] which already exists", partitionName);
```

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**